compiler/testData/codegen/box/callableReference/arrayConstructor.kt

```diff
@@ -0,0 +1,6 @@
+// WITH_RUNTIME
+// IGNORE_BACKEND: JS_IR
```

Comment:
🤔

Reply:
Same error as in my other PR related to array constructors - https://youtrack.jetbrains.com/issue/KT-46520